src/utils.ts

```diff
@@ -126,14 +117,17 @@ export const delay = (ms: number): Promise<void> => new Promise(res => setTimeou
 export enum PondVersions {
   Version1 = 1,
   Version2 = 2,
+  Version3 = 3,
 }
-export const defaultPondVersions = PondVersions.Version2
+export const defaultPondVersions = PondVersions.Version3
 export const parsePondVersion = (version: string): PondVersions => {
```

Comment:
Since help explicitly specifies `-v` can be `[1,2,3]`, we should check for that. Currently, it accepts e.g. `2.1.23` and correctly uses `2` then, but it might be confusing. Better reject it right away.